extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -384,23 +383,23 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 			.withProps({
 				columns: [
 					{
-						displayName: constants.SOURCE_DATABASE,
-						valueType: azdata.DeclarativeDataType.string,
+						displayName: '',
```

Comment:
Is displayName a mandatory parameter? Can we skip it since it is empty string?

Reply:
@sahuroshan  Yes it is a mandatory field, so cannot skip this.